_posts/blog/2019-08-01-traceroute-bq-newdata-available.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: blog
+title: "New Traceroute Table Available to public"
+author: "Ya Chang"
+date: 2019-08-01
+breadcrumb: blog
+categories:
+  - data
+  - traceroute
```

Comment:
Adding a the tag, `traceroute` requires a new category page in `/_pages/categories/`